doc/wg/network/notes/network-notes-2023-08-10.md

```diff
@@ -0,0 +1,96 @@
+# Tock Network Notes 2023-08-10
+
+## Attendees
+ * Branden Ghena
+ * Alexandru Radovici
+ * Felix Mada
+ * Leon Schuermann
+ * Cristian Rusu
+ * Tyler Potyondy
+
+# Introduction
+ * Alex: First meeting! Goal to define how networking is going to work in Tock. Round of introductions first!
+ * Felix: Working at OxidOS, mainly on CAN bus. Started with Tock a couple of months ago. Worked on embedded for four-five years.
+ * Leon: PhD student at Princeton working with Amit Levy. Working on Tock for five years. Funny enough my first real project was an Ethernet driver for Tock. Still in progress...
+ * Tyler: Master's student at UC San Diego working with Pat Pannuto. Only involved in Tock for a few months. Working on adding support for Thread networking: 802.15.4, 6lowpan, UDP, etc.
+ * Cristian: I teach at Bucharest. Background in wireless communication: 5G stuff. I'm here because I know about Tock and networking.
```

Comment:
```suggestion
 * Cristian: I teach at University of Bucharest. Background in wireless communication: 5G stuff. I'm here because I know about Tock and networking.
```